apps/src/code-studio/pd/application/teacher/TeacherApplication.jsx

```diff
@@ -123,11 +124,16 @@ const TeacherApplication = props => {
     />
   );
 };
+
 TeacherApplication.propTypes = {
   ...FormController.propTypes,
   accountEmail: PropTypes.string.isRequired,
   userId: PropTypes.number.isRequired,
   schoolId: PropTypes.string
 };
 
+TeacherApplication.defaultProps = {
+  sessionStorage: sessionStorage
```

Comment:
Another option is to call this prop `storage` to differentiate it from `sessionStorage` but requires change `FormController` as well ... thoughts?